web/src/LogActions.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { mount } from "enzyme"
+import React from "react"
+import {
+  LogFontSizeScaleCSSProperty,
+  LogFontSizeScaleLocalStorageKey,
+  LogFontSizeScaleMinimumPercentage,
+  LogsFontSize,
+} from "./LogActions"
+
+describe("LogsFontSize", () => {
+  const cleanup = () => {
+    localStorage.clear()
+    document.documentElement.style.removeProperty("--log-font-scale")
+  }
+
+  beforeEach(() => {
+    cleanup()
+    // CSS won't be loaded in test context, so just explicitly set it
+    document.documentElement.style.setProperty("--log-font-scale", "100%")
+  })
+  afterEach(cleanup)
+
+  const getCSSValue = () =>
+    getComputedStyle(document.documentElement).getPropertyValue(
+      LogFontSizeScaleCSSProperty
+    )
+  // react-storage-hooks JSON (de)serializes transparently,
+  // need to do the same when directly manipulating local storage
+  const getLocalStorageValue = () =>
+    JSON.parse(localStorage.getItem(LogFontSizeScaleLocalStorageKey) || "")
+  const setLocalStorageValue = (val: string) =>
+    localStorage.setItem(LogFontSizeScaleLocalStorageKey, JSON.stringify(val))
+
+  it("restores persisted font scale on load", () => {
+    setLocalStorageValue("360%")
+    mount(<LogsFontSize />)
+    expect(getCSSValue()).toEqual("360%")
+  })
+
+  it("decreases font scale", () => {
+    const root = mount(<LogsFontSize />)
+    root.find("LogFontSizeDecreaseButton").simulate("click")
```

Comment:
i think the idiomatic way to do this is to import the styled component itself, and do a .find(Component) (rather than passing around strings)

Reply:
For whatever reason this would NOT work yesterday but yarn is happy now - fixed in ee8fdff68f7a1b32e6bc64bebf9fa55ead759dfe